app/decorators/models/pseudonym_decorator.rb

```diff
@@ -1,6 +1,7 @@
 Pseudonym.class_eval do
   after_commit -> { PipelineService::V2.publish self }
   after_save :update_identity_mapper?, if: :integration_id_changed?
+  validates :integration_id, uniqueness: true, :allow_blank => true
```

Comment:
Question about the validates:

Does validating the integration_id and allow_blank => true allow for opportunities to write a null field to the db?